dracut/repartition/endless-repartition.sh

```diff
@@ -29,8 +29,15 @@
 #
 # Based on code from dracut-modules-olpc.
 
+root_part=$(systemctl show -p What sysroot.mount)
```

Comment:
Clever. Does this work early in the initramfs? Does sysroot.mount get generated with the root= argument? I'm asuming you tested this.